hgeometry/src/Algorithms/Geometry/LineSegmentIntersection/BentleyOttmann.hs

```diff
@@ -10,7 +10,13 @@
 -- and Ottmann.
 --
 --------------------------------------------------------------------------------
-module Algorithms.Geometry.LineSegmentIntersection.BentleyOttmann where
+module Algorithms.Geometry.LineSegmentIntersection.BentleyOttmann
+  ( intersections
+  , interiorIntersections
+    -- FIXME: Move ordAt and xCoordAt to Data.Geometry.LineSegment?
```

Comment:
I guess we can indeed move ordAt and xCoordAt to LineSegment. (Somehow those functions feel a bit on the specific side there, but on the other hand I importing this module for them is also slightly strange I guess)

Reply:
I'll leave those functions for now until we can figure out a good home for them.